src/multiworld.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+#ifndef CATA_SRC_MULTIWORLD_H
+#define CATA_SRC_MULTIWORLD_H
+
+#include <climits>
+#include <cstdint>
+#include <iosfwd>
+#include <new>
+#include <optional>
+#include <string>
+#include <vector>
+
+#include "game.h"
+#include "calendar.h"
+#include "catacharset.h"
+#include "color.h"
+#include "damage.h"
+#include "translations.h"
+#include "type_id.h"
+#include "point.h"
+class multiworld
```

Comment:
My biggest piece of feedback is that this class shouldn't exist, get_world_prefix(), travel_to_world(), and the master world_prefix member should just live in the game class.  It's already (necessarily) fiddling with a ton of stuff the game class owns, just move it into the game class.

In a broader sense, yes the game class is too big and OTHER things should be migrated out of it, but this kind of top-level coordination is precisely the domain of the game class.